src/store/slices/playlists/slice.ts

```diff
@@ -52,7 +73,16 @@ const slice = createSlice({
         },
         [createPlaylist.rejected.type]: (state) => {
             state.loading.createPlaylistLoading = false
-        }
+        },
+        [getCurrentUserPlaylists.rejected.type]: (state) => {
+            state.loading.getPlaylists = false
+        },
+        [getCurrentUserPlaylists.fulfilled.type]: (state) => {
+            state.loading.getPlaylists = false
+        },
+        [getCurrentUserPlaylists.pending.type]: (state) => {
+            state.loading.getPlaylists = true
+        },
```

Comment:
Could you pls. add tests for these 3 state changes?

Reply:
i deleted these extrareducers